bin/cheribsdtest/cheribsdtest_vm.c

```diff
@@ -1820,6 +1824,7 @@ CHERIBSDTEST(cheri_revoke_async_fork,
 
 	cheribsdtest_success();
 }
+#endif
```

Comment:
I added a `/* CHERIBSD_THREAD_TESTS */` here in the version I'd not pushed yet.

Reply:
Oh sure, that's probably helpful here